client/app/components/queries/ScheduleDialog.jsx

```diff
@@ -21,21 +22,24 @@ const { Option, OptGroup } = Select;
 export class ScheduleDialog extends React.Component {
   static propTypes = {
     show: PropTypes.bool.isRequired,
-    // eslint-disable-next-line react/forbid-prop-types
-    query: PropTypes.object.isRequired,
+    schedule: RefreshScheduleType,
```

Comment:
I like it that it takes a `schedule` object now. Will become handy when we upgrade the dashboard scheduler or when we introduce scheduling for alerts which is separate from the query. 👌